hscontrol/grpcv1.go

```diff
@@ -671,6 +674,73 @@ func (api headscaleV1APIServer) DeleteApiKey(
 	return &v1.DeleteApiKeyResponse{}, nil
 }
 
+func (api headscaleV1APIServer) GetPolicy(
+	_ context.Context,
+	_ *v1.GetPolicyRequest,
+) (*v1.GetPolicyResponse, error) {
+	switch api.h.cfg.Policy.Mode {
+	case types.PolicyModeDB:
+		p, err := api.h.db.GetPolicy()
+		if err != nil {
+			return nil, err
+		}
+
+		// We retain the HuJSON format of the policy while storing
+		// it in the database. But should we return the same in the
+		// get policy response? Or should we consider returning the
+		// policy in JSON format?
+
+		return &v1.GetPolicyResponse{
+			Policy:    p.Data,
+			UpdatedAt: timestamppb.New(p.UpdatedAt),
+		}, nil
+	case types.PolicyModeFile:
+		b, err := json.Marshal(api.h.ACLPolicy)
```

Comment:
I suppose if we return the content of the file instead of the loaded content, we could preserve comments.

Reply:
Since we support both YAML and HuJSON, the response may differ based on the file format used for the ACLs.

I'm updating this block to return the contents of the file. But we can consider dropping YAML support altogether. What do you think?

Reply:
I'm open for that yes

Reply:
Sounds good. I'll go ahead and remove YAML support.